roles/get_image_hash/tasks/main.yml

```diff
@@ -68,24 +68,24 @@
   when: get_release_images | bool
   block:
     - name: Update released items
-      vars: 
+      vars:
         updated_release_images: []
       set_fact:
-        updated_release_images: "{{ updated_release_images + [item | combine({'url': item.url.split(':')[0] + '@' + image_hashes['release_' + item.version + '_' + item.cpu_architecture]  })] }}"
-      when: 
+        updated_release_images: "{{ updated_release_images + [item | combine({'url': item.url.split('@')[0] + '@' + image_hashes['release_' + item.version + '_' + item.cpu_architecture]  })] }}"
```

Comment:
Ah I this was to replace the tag with the hash. 
Looks like we need a check to make sure the url doesn't already contain the hashed version.

Reply:
gotcha, I'll close this PR then